File: cpp/suma_parzyste.cpp
/*
 * suma_parzyste.cpp
 *
 */


#include <iostream>

using namespace std;

int main(int argc, char **argv)
{
	int suma, liczba;
	int start, koniec;
	start = koniec = 0;
	
	cout << "Podaj liczbę począkową: ";
	cin >> start;
	
	cout << "Podaj liczbę końcową: ";
	cin >> koniec;
	
	suma = 0;
	
	for (liczba = start; liczba < koniec+1; liczba++) {
		if (liczba % 2 == 0)
			suma = suma + liczba;
	}
	cout << suma;
	return 0;
}

File: python/suma_parzyste.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  suma_parzyste.py
#

def main(args):
    poczatek = int(input("Podaj liczbę początkową: ")) 
    koniec = int(input("Podaj liczbę końcową: ")) 
    suma = 0
    for liczba in range(poczatek, koniec + 1):
        if liczba % 2 == 0:
            suma = suma + liczba
    print(suma)
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
    
